contrib/init/systemd/README.md

```diff
@@ -3,20 +3,37 @@
 > NOTE: These instructions are valid for Ubuntu versions using `systemd` init
 > system.
 
-Run the following commands:
+## TL;DR
+1. Get the latest [release](https://github.com/Azure/azurefile-dockervolumedriver/releases)
+2. Put the binary into `/usr/bin/azurefile-dockervolumedriver`
+3. Get the .default and .service files from the .tar.gz file and deploy them
+4. Reload systemd
+
+## In-depth walkthrough
+To ssh onto machines created with the `docker-machine` commands, you'll probably need to ssh via `docker-machine ssh [machinename]` to perform the following.
 
 0. `sudo -s`
-0. Download the binary from “Releases” tab of the repo to `/opt/bin/`: `wget -qO/usr/bin/azurefile-dockervolumedriver [url]`
-0. Make it executable: `chmod +x /usr/bin/azurefile-dockervolumedriver`
+0. Download the tar.gz from "Releases" tab of the repo
+    + Currently this would be `wget https://github.com/Azure/azurefile-dockervolumedriver/archive/0.2.1.tar.gz` 
+0. Extract and copy the extracted files to `/usr/bin/azurefile-dockervolumedriver`: 
+    + Decompress the archive:  `tar -xvf 0.2.1.tar.gz`
+0. Download the binary from the "Releases" tab of the repo to `/usr/bin/azurefile-dockervolumedriver`
+    + Use wget: `wget -qO/usr/bin/azurefile-dockervolumedriver https://github.com/Azure/azurefile-dockervolumedriver/releases/download/0.2.1/azurefile-dockervolumedriver`
+    + Make it executable `chmod +x /opt/bin/azurefile-dockervolumedriver`
 0. Save the `.default` file to `/etc/default/azurefile-dockervolumedriver`
+    + Copy the file: `cp azurefile-dockervolumedriver-0.2.1/contrib/init/systemd/azurefile-dockervolumedriver.default /etc/default/azurefile-dockervolumedriver`
 0. Edit `/etc/default/azurefile-dockervolumedriver` with your Azure Storage Account credentials.
+    + Edit in vi (or another editor): `vi /etc/default/azurefile-dockervolumedriver`
+    + Use ESC, :, x, ENTER to save an exit if you use vi
 0. Save the `.service` file to `/etc/systemd/system/azurefile-dockervolumedriver.service`
+    + Make the requisite directories if they don't exist: `mkdir /etc/systemd && mkdir /etc/systemd/system`
```

Comment:
The default image used by docker-machine for azure is 15.04 o_O It's also the one MSFT sign off the ODBC 13 driver against. I think that's a nuts decision, but I'm not gonna argue with them all who made those decisions! It was just another hiccup needing to be taken into account through the "common" path if one used Docker docs to get started, as opposed to the Docker image in Azure.